app/src/main/java/ch/epfl/polybazaar/UI/SalesOverview.java

```diff
@@ -47,15 +47,12 @@ protected void onCreate(Bundle savedInstanceState) {
         // Create adapter passing in the sample LiteListing data
         adapter = new LiteListingAdapter(liteListingList);
 
-        adapter.setOnItemClickListener(new LiteListingAdapter.OnItemClickListener() {
-            @Override
-            public void onItemClick(View view) {
-                int viewID = view.getId();
-                String listingID = adapter.getListingID(viewID);
-                Intent intent = new Intent(SalesOverview.this, SaleDetails.class);
-                intent.putExtra("listingID", listingID);
-                startActivity(intent);
-            }
+        adapter.setOnItemClickListener(view -> {
+            int viewID = view.getId();
+            String listingID = adapter.getListingID(viewID);
+            Intent intent = new Intent(SalesOverview.this, SaleDetails.class);
+            intent.putExtra("listingID", listingID);
```

Comment:
I know this is not part of your PR, but we should consider changing the "listingID" string to a static variable later

Reply:
Yes, done.